#! /bin/sh /usr/share/dpatch/dpatch-run
## api.dpatch by Samuel Mimram <smimram@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Correct links in the documentation, see #423590.

@DPATCH@
diff -urNad libsamplerate-0.1.2~/doc/api.html libsamplerate-0.1.2/doc/api.html
--- libsamplerate-0.1.2~/doc/api.html	2004-09-12 23:11:14.000000000 +0200
+++ libsamplerate-0.1.2/doc/api.html	2007-05-22 22:01:21.000000000 +0200
@@ -74,13 +74,13 @@
 The API allows three methods for accessing the capabilies of the library:
 </P>
 <UL>
-	<LI>A <A HREF="api_simple.html">simple interface</A> which can sample rate convert 
+	<LI>A <A HREF="http://www.mega-nerd.com/SRC/api_simple.html">simple interface</A> which can sample rate convert 
 		a single block of samples (one or more channels) in one go.
 		The simple API is less capable than the full API.
-	<LI>A <A HREF="api_full.html">more fully featured interface</A> which allows time
+	<LI>A <A HREF="http://www.mega-nerd.com/SRC/api_full.html">more fully featured interface</A> which allows time
 		varying sample rate conversion on streaming data (again one or more
 		channels).
-	<LI>A <A HREF="api_callback.html">callback interface</A> which has the same 
+	<LI>A <A HREF="http://www.mega-nerd.com/SRC/api_callback.html">callback interface</A> which has the same 
 		functionality as the interface above but allows the details of input and
 		output to be separated.
 		The output is generated by call a read function and the library calls a user
@@ -92,15 +92,15 @@
 The parts of the API which are common to both are:
 </P>
 <UL>
-	<LI> The <A HREF="api_misc.html#ErrorReporting">error reporting</A>  mechanisim.
-	<LI> The available <A HREF="api_misc.html#Converters">converter</A> types.
-	<LI> The <A HREF="api_misc.html#SRC_DATA">SRC_DATA</A> struct.
+	<LI> The <A HREF="http://www.mega-nerd.com/SRC/api_misc.html#ErrorReporting">error reporting</A>  mechanisim.
+	<LI> The available <A HREF="http://www.mega-nerd.com/SRC/api_misc.html#Converters">converter</A> types.
+	<LI> The <A HREF="http://www.mega-nerd.com/SRC/api_misc.html#SRC_DATA">SRC_DATA</A> struct.
 </UL>
 <P>
 All three versions of the API are restricted to operating on buffers of ISO C 
 Standard  <B>float</B> data. 
 However, there are two 
-	<A HREF="api_misc.html#Aux">auxillary functions</A>
+	<A HREF="http://www.mega-nerd.com/SRC/api_misc.html#Aux">auxillary functions</A>
 for converting arrays of float data to and from short data.
 </P>
 
